src/templates/components/time.html

```diff
@@ -14,7 +14,4 @@
   ng-pattern="component.validate.pattern"
   custom-validator="component.validate.custom"
   ng-attr-placeholder="{{ component.placeholder | formioTranslate:null:options.building }}"
-  ui-mask="{{ component.inputMask }}"
```

Comment:
you did not add ```formio-mask``` to this. Don't think it matters since nobody uses this component type anyway, but it should still be fixed.

Reply:
inputMask isn't settable on the time component so it is safe to remove it. Was probably just added as cruft when creating the time component.

Reply:
ok then.